Day-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,44 @@
 /* All styles for your header Components live here */
+headerImage
+headerTitleAndText
+headerText
+headerLogo
+titleBox
+titleUserName
+titleHandle
+titleDate */
+ .headerBox{
+    display: flex;
+  
+}
+.headerTitleAndText{
```

Comment:
CSS specificity still apply with React, Just keep in mind how you're having it set-up in your component.